css/slather.css

```diff
@@ -0,0 +1,104 @@
+/* --------------------------------------------------------
```

Comment:
I don't know where to put the css file, so I put it in the root folder. And all of the generated html will link to this css from the gem's root folder.

I grab the colour scheme from the logo, and 'slather' it to the stylesheet whenever appropriate. I've used coverall's HTML/CSS as a reference.